src/PushmixChannel.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace NotificationChannels\Pushmix;
+
+use GuzzleHttp\Exception\RequestException;
+use Illuminate\Notifications\Notification;
+use NotificationChannels\Pushmix\Exceptions\CouldNotSendNotification;
+
+class PushmixChannel
+{
+    /** @var PushmixClient */
+    protected $pusmixClient;
+
+    public function __construct(PushmixClient $pusmixClient)
+    {
+        $this->pusmixClient = $pusmixClient;
+    }
+
+    /**
+     * Send Notification.
+     *
+     * @param mixed $notifiable
+     * @param \Illuminate\Notifications\Notification $notification
+     *
+     * @return \Psr\Http\Message\ResponseInterface
+     * @throws \NotificationChannels\Pushmix\Exceptions\CouldNotSendNotification
+     */
+    public function send($notifiable, Notification $notification)
+    {
+        if (! $to = $notifiable->routeNotificationFor('Pushmix')) {
```

Comment:
```suggestion
        if (! $to = $notifiable->routeNotificationFor('Pushmix')) {
```

lowercase for routing